### PR TITLE
Rename playground to web in the UI and docs

### PR DESCRIPTION
**Pull request by carlosms**
Part of #218.

The changes in this PR do not depend on releases or repository names, and can be merged at any time.

Sorry @ricardobaeta, the styling of the name in the sidebar (_playgroun{d}_) will be lost!

### CODE REVIEW

**Comment by bzz** [comment]
Overall looks great to me, may be as @eiso mentioned in https://github.com/bblfsh/web/issues/149#issuecomment-413840354 it can also be referred as `gitbase web client` from time to time

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-[![Build Status](https://travis-ci.org/src-d/gitbase-playground.svg)](https://travis-ci.org/src-d/gitbase-playground)
-[![codecov.io](https://codecov.io/github/src-d/gitbase-playground/coverage.svg)](https://codecov.io/github/src-d/gitbase-playground)
+[![Build Status](https://travis-ci.org/src-d/gitbase-web.svg)](https://travis-ci.org/src-d/gitbase-web)
+[![codecov.io](https://codecov.io/github/src-d/gitbase-web/coverage.svg)](https://codecov.io/github/src-d/gitbase-web)
```
Comment (bzz) [inline README.md]: These seems to depend on repo name, but are ofc fine to break

File: README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+Application to query Git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase), it allows to perform SQL queries on the Git history and the [Universal AST](https://doc.bblf.sh/) of the code itself.
 
 ![Screenshot](.github/screenshot.png?raw=true)
 
 # Usage
 
 ## With Docker Compose
 
-The easiest way to run Gitbase Playground and its dependencies is using [Docker Compose](https://docs.docker.com/compose/install/).
+The easiest way to run Gitbase Web and its dependencies is using [Docker Compose](https://docs.docker.com/compose/install/).
```
Comment (bzz) [inline README.md]: web client?

File: README.md [hunk 3]
```diff
@@ -41,29 +41,29 @@ for cleanup.
 
 ## Without Docker Compose
 
-The playground will run the queries against a [gitbase](https://docs.sourced.tech/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server. Make sure both are properly configured.
+The application will run the queries against a [gitbase](https://docs.sourced.tech/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server. Make sure both are properly configured.
 
-Then you can choose to run the Gitbase Playground either as a docker container, or as a binary.
+Then you can choose to run the Gitbase Web either as a docker container, or as a binary.
```
Comment (bzz) [inline README.md]: web client

File: README.md (hunk 1 above)
Comment (carlosms) [reply README.md]: Yes, those will show as gray boxes, but it should still render.

File: README.md (hunk 3 above)
Comment (carlosms) [reply README.md]: Done in b9f39e7.

**Pull request merged**